logstage/logstage-core/src/main/scala-3/izumi/logstage/api/LogValuesMacro.scala

```diff
@@ -0,0 +1,55 @@
+package izumi.logstage.api
+
+import izumi.fundamentals.platform.language.CodePositionMaterializer
+import izumi.logstage.api.Log.{Level, Message}
+import izumi.logstage.api.logger.{AbstractLogIO, AbstractLogger}
+
+import scala.annotation.tailrec
+import scala.quoted.*
+
+object LogValuesMacro {
+  def logValuesIO[F[_]: Type](
+    using Quotes
+  )(logger: Expr[AbstractLogIO[F]],
+    level: Expr[Level],
+    values: Expr[Seq[Any]],
+  ): Expr[F[Unit]] = {
+    val messageString = createMessageString(values)
+    '{ $logger.log($level)(Message($messageString))(CodePositionMaterializer.materialize) }
```

Comment:
Here you may also call Message macro (and CodePosition macro) within the current macro expansion:

```suggestion
    '{ $logger.log($level)(${ LogMessageMacro.message(messageString, strict = false) $})(${ CodePositionMaterializerMacro.getCodePositionMaterializer() }) }
```